ers/nats_it_test.go

```diff
@@ -150,8 +150,8 @@ var natsCfg string = `{
 			"id": "nats_reader",
 			"type": "*nats_json_map",
 			"source_path": "%s",
-			"ees_success_ids": ["nats_processed"],
-			"flags": ["*dryrun"],
+			"ees_success_ids": [],
+			"flags": ["*dryrun", "*export:nats_processed"],
```

Comment:
This approach does not look like achieving the old behavior with success/fail ids.